pennylane_qiskit/basic_aer.py

```diff
@@ -54,14 +57,56 @@ class BasicAerDevice(QiskitDevice):
 
     def __init__(self, wires, shots=1024, backend="qasm_simulator", **kwargs):
 
-        max_ver = Version("0.45.3")
+        max_ver = Version("0.46", partial=True)
 
         if Version(qiskit.__version__) > max_ver:
             raise RuntimeError(
-                f"The devices in the PennyLane Qiskit plugin are currently only compatible "
-                f"with versions of Qiskit below 0.46. You have version {qiskit.__version__} "
-                f"installed. Please downgrade Qiskit to use the devices. The devices will be "
-                f"updated in the coming weeks to be compatible with Qiskit 1.0!"
+                f"Qiskit has discontinued the BasicAer device, so it can only be used in"
+                f"versions of Qiskit below 1.0. You have version {qiskit.__version__} "
+                f"installed. For a Python simulator, use the 'qiskit.basicsim' device "
+                f"instead. Alternatively, you can downgrade Qiskit to use the "
+                f"'qiskit.basicaer' device."
             )
 
         super().__init__(wires, provider=qiskit.BasicAer, backend=backend, shots=shots, **kwargs)
+
+
+class BasicSimulatorDevice(QiskitDevice):
+    """A PennyLane device for the native Python Qiskit simulator from BasicProvider.
+
+    For more information on the BasicSimulator backend options and transpile options, please visit the
+    `BasicProvider documentation <https://docs.quantum.ibm.com/api/qiskit/providers_basic_provider>`_.
+
+    A range of :code:`backend_options` that will be passed to the simulator and
+    a range of transpile options can be given as kwargs.
+
+    Args:
+        wires (int or Iterable[Number, str]]): Number of subsystems represented by the device,
+            or iterable that contains unique labels for the subsystems as numbers (i.e., ``[-1, 0, 2]``)
+            or strings (``['aux_wire', 'q1', 'q2']``).
```

Comment:
Here we still need `wires` because we're using the old device API?

Reply:
yes!